### ISSUE TITLE
Feature request: Filter in the timer view

### ISSUE THREAD

**Issue by smatthie** [mtierltd/timetracker#63]
Can you add a filter to the timer view, similar to the filters in the report view?

The reason is that I usually restart an earlier task rather than a new one where I then have to set Name, Project, tags etc. every time. But for projects that are on hold for a while I then have to scroll down very far to find their old entries.

**Comment by LukeLR** [comment]
+1, I can see this is already being worked on, at least I can already see a filter when using the timer view. But at least for me, it isn't functional yet, e.g. when I select "yesterday", entries from today are still visible. So I guess this is still under development.

But I think this is a really useful feature, as it seems like all entries are displayed otherwise? This will probably slow down the page in the future, I can already notice increased loading times with only two days of entries.

But thanks a lot for your great work, this is an awesome app and I like where this is going :) I actually prefer it over https://apps.nextcloud.com/apps/timemanager.

**Comment by smatthie** [comment]
Yes, there's a filter for time range, but as you say, it doesn't work (properly); there was a separate bug #56 for this, but this was closed; not sure why (perhaps the fix has not been pushed to the app store yet).
I meant particularly filters for projects etc.

**Comment by smatthie** [comment]
Are any thoughts on this?
I increasingly have the problem that I want to continue an older activity, and finding it in the long list of entries is really cumbersome. It would be very helpful to be able to filter the list in the timer view by project name.
